README.md

```diff
@@ -52,28 +52,11 @@ When you believe your ECIP is mature and ready to progress past the draft phase,
 * **Rejected** - Reasons for rejecting ECIPs include duplication of effort, disregard for formatting rules, unfocused or too broad, being technically unsound, not providing proper motivation, or obvious popular disapproval.
 * **Withdrawn** - ECIP authors may decide to change the status between Draft, Deferred, or Withdrawn. The ECIP editor may also change the status to Deferred if no progress is being made on the ECIP.
 
-# Important ECIPs
+## ECIP List
 
-## Accepted ECIPs
+See [ECIPs website](https://ecips.ethereum.org) for the complete list of ECIPs.
```

Comment:
Fix to `https://ecips.ethereumclassic.org`

Reply:
Addressed.